src/core/mainoptions.cpp

```diff
@@ -168,11 +168,16 @@ Options for scripting or debugging:
                                PATH
         --extract-result PATH  extract file data stored in result file at PATH
                                to the current working directory
+    -f, --format FORMAT        specify format for --print-log-stats and
```

Comment:
Since this is used with a long option anyway, and typically only from other programs (not interactively), I suggest removing the short `-f` form. That would save `-f` for future options that might be more commonly used than specifying the format.

Reply:
Agreed!